build.sh

```diff
@@ -35,37 +37,35 @@ tcStatus() {
   tcMsg "buildStatus status='$status' text='$1'"
 }
 
-tcLog 'Show info'
-java -version
-mvn -version
-node --version
-npm --version
+tcLog "Show info (processors=$processors parallel=$parallel)"
+type java && java -version
+type mvn && mvn -version
+type node && node --version
+type npm && npm --version
+type pnpm && pnpm --version
 uname -a
 
 cmd="npm install --silent --quiet --no-progress"
 tcLog "Install NPM packages - $cmd"
 $cmd || exit 1
 
-cmd="scripts/mergeITs.js "`echo $elements`
-tcLog "Merge IT modules - $cmd"
-$cmd || exit 1
-
-cmd="mvn install -DskipTests -Drelease -B -T C$processors"
-tcLog "Installing flow components - $cmd"
-$cmd || exit 1
-
-if [ -n "$BUILD" ]
+cmd="mvn install -Drelease -B -q -T C$processors"
+tcLog "Unit-Testing and Installing flow components - $cmd"
+sleep 10
+$cmd
+if [ $? != 0 ]
 then
-  cmd="mvn test -B -Drun-it -T C$processors"
-  tcLog "Unit-Testing - $cmd"
-  $cmd
+  tcLog "Unit-Testing and Installing flow components (2nd try) - $cmd"
+  sleep 30
```

Comment:
What are we waiting 30 seconds for?

Reply:
As said above, without sleep the second time can fail, but the sleep makes the build pass